iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/primary/processing/oozie_app/workflow.xml

```diff
@@ -711,48 +711,26 @@
 	<!-- start of documents classification part -->
 	<decision name="decision-documentsclassification">
 		<switch>
-			<case to="transformers_documentsclassification">${active_documentsclassification eq "true"}</case>
+			<case to="documentsclassification">${active_documentsclassification eq "true"}</case>
 			<default to="skip-documentsclassification" />
 		</switch>
 	</decision>
 
-	<action name="transformers_documentsclassification">
+	
+	<action name="documentsclassification">
 		<sub-workflow>
-			<app-path>${wf:appPath()}/transformers_documentsclassification</app-path>
+			<app-path>${wf:appPath()}/documentsclassification</app-path>
```

Comment:
You could remove priming for old transformer workflow in import.txt.
And definitely you should modify documentsclassification_main workflow to documentsclassification as you changed it:
`documentsclassification_main classpath eu/dnetlib/iis/workflows/documentsclassification/main/oozie_app`